scripts/test.sh

```diff
@@ -353,9 +353,20 @@ case "$TARGET" in
 
   check_test_results ReactTests
   ;;
+
+"node-test-runners")
```

Comment:
Should this be "test-runners"?
Is this different from https://github.com/realm/realm-js/blob/master/scripts/test.sh#L397-L401?

Reply:
I'll remove it and adjust `Jenkinsfile`.